### ISSUE TITLE
Faction Leader incorrect drops

### ISSUE THREAD

**Issue by ghost** [Wall-core/Everlook-Bugtracker#8]
I have killed all faction leaders and they drop incorrect gold and some of them don't award honor.

Horde:
- Vol'jin(10540) ------------------- no gold - 488 honor
- Thrall(4949) ------------------------ 6s 8c - 488 honor
- Cairne Bloodhoof(3057) ---------- 2s 57c - 488 honor
- Sylvanas(10181) ------------------- 6s 2c - 488 honor
- Varimathras(2425) -------------- no gold - 488 honor

Aliance:
- High Tinker Mekkatorque(7937) ---- 16c - 488 honor
- Magni Bronzebeard(2784) ------- 4s 96c - 488 honor
- Highlord Bolvar(1748) ----------- 4s 70c - no honor
- Trynade Whisperwind(7999) ----- 8s 64c - no honor
- Arch Druid Fandral(3516) ------- no gold - 488 honor
- Jaina(4968) ---------------- 70g 55s 36c - no honor
- Archbishop Benedictus(1284) --- no gold - 488 honor

Every faction leader should award honor and more gold (about the same gold as Jaina)

**Comment by Wall-core** [comment]
I don't believe that Jaina or Bolvar are intended to give honor. I can't find confirmation of this, but I'm fairly certain that only Darnassus and Undercity have 2 npcs marked as leaders
